test/export/test_export.py

```diff
@@ -90,6 +95,7 @@ def conflicting_constraints(x):
         ):
             ep(torch.tensor([3]))
 
+    @testing.expectedFailureNonStrict
```

Comment:
It seems bad that we are leaking non-strict info into this test file, could we maybe track non-strict failures in the non_strict test file instead? I imagine once we have seperate files for "retrace", "ser_der", we can't be applying decorators on this file. One way you can do is you can create a list in "test_export_non_strict" file and make sure that each entry in that list fails.

Reply:
> I imagine once we have seperate files for "retrace", "ser_der", we can't be applying decorators on this file.

Actually, we can stack the decorators and it's what I do in the PR stack. 

> One way you can do is you can create a list in "test_export_non_strict" file and make sure that each entry in that list fails.

That's another approach and seems good in this PR. 

I regard a test unit as a typical usage from user's perspective. If we consider all the test variations (non-strict, retrace, ser_der), centralized marks (decorators) close to the test case could clearly show the view of support power for each test unit. If we scatter those marks into each variation, it's hard to identify the existing problems of the each test unit.

Reply:
> > I imagine once we have seperate files for "retrace", "ser_der", we can't be applying decorators on this file.
> 
> Actually, we can stack the decorators and it's what I do in the PR stack.
> 
> > One way you can do is you can create a list in "test_export_non_strict" file and make sure that each entry in that list fails.
> 
> That's another approach and seems good in this PR.
> 
> I regard a test unit as a typical usage from user's perspective. If we consider all the test variations (non-strict, retrace, ser_der), centralized marks (decorators) close to the test case could clearly show the view of support power for each test unit. If we scatter those marks into each variation, it's hard to identify the existing problems of the each test unit.



> Actually, we can stack the decorators and it's what I do in the PR stack.

Oh I meant from UX perspective.

Reply:
> I regard a test unit as a typical usage from user's perspective. If we consider all the test variations (non-strict, retrace, ser_der), centralized marks (decorators) close to the test case could clearly show the view of support power for each test unit. If we scatter those marks into each variation, it's hard to identify the existing problems of the each test unit.

I see, that's fair. @zhxchen17 @angelayi  what do u guys think?